test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -76,6 +76,10 @@
     ("aten::linalg_outer.out", datetime.date(2020, 8, 30)),
     ("aten::_compute_linear_combination", datetime.date(2020, 9, 1)),
     ("__getstate__", datetime.date(2020, 9, 1), "Conv[23]dPackedParams"),
+    ("aten::linspace", datetime.date(2020, 9, 30)),
```

Comment:
do you know why these trigger?  It should be safe from a serialization perspective.

Reply:
I added them preemptively without verifying they would trigger. I don't know if they would.